Makefile

```diff
@@ -0,0 +1,31 @@
+setup: setup-rfid setup-contract
+
+.PHONY: setup-rfid
+setup-rfid:
+	@curl -i http://localhost:9410/api/v0/token -H 'content-type: application/json' -d '{\
+	  "countryCode": "GB",\
+	  "partyId": "TWK",\
+	  "type": "RFID",\
+	  "uid": "DEADBEEF",\
+	  "contractId": "GBTWK012345678V",\
+	  "issuer": "Thoughtworks",\
+	  "valid": true,\
+	  "cacheMode": "ALWAYS"\
+	}'
+
+.PHONY: setup-contract
+setup-contract:
+	@curl -i http://localhost:9410/api/v0/token -H 'content-type: application/json' -d '{\
+	"countryCode": "GB",\
+	"partyId": "TWK",\
+	"type": "RFID",\
+	"uid": "EMP77TWTW99999",\
+	"contractId": "GBTWK012345678V",\
+	"issuer": "Thoughtworks",\
+	"valid": true,\
+	"cacheMode": "ALWAYS"\
+	}'
+
+.PHONY: debug
+debug:
+	@curl -i http://localhost:9410/api/v0/cs/cs001 -H 'content-type: application/json' -d '{"securityProfile":2}' || docker-compose restart lb
```

Comment:
1. I don't understand why the lb is being restarted. It does not need to be restarted for the security profile change to be applied.
2. docker-compose is a deprecated program and may not exist on systems that have a perfectly good docker compose environment - see https://docs.docker.com/compose/migrate/. This is why other scripts work out whether or not `docker compose` is available and only fall back to using `docker-compose`